docs/en/reference/sql/dql/SELECT_STATEMENT.md

```diff
@@ -111,27 +111,21 @@ TableAsName
-| [`WINDOW` Clause](../dql/WINDOW_CLAUSE.md)     | Online Training not supported | 
-The window clause is used to define one or several windows. Windows can be named or anonymous. Users can call aggregate functions on the window to perform some analytical calculations (```sql agg_func() over window_name```). During Online Serving, you need to follow [OP's usage specification under Online Serving](../deployment_manage/ONLINE_SERVING_REQUIREMENTS.md) |
-| [`LIMIT` Clause](../dql/LIMIT_CLAUSE.md)       | Online Serving does not support   | The Limit clause is used to limit the number of results. OpenMLDB currently only supports Limit accepting one parameter, indicating the maximum number of rows of returned data; |
-| `ORDER BY` Clause                              | not supported                 | Standard SQL also supports the OrderBy clause. OpenMLDB does not currently support the Order clause. For example, the query `SELECT * from t1 ORDER BY col1;` is not supported in OpenMLDB. |
+| `SELECT` Statement Elements                    | Offline Mode | Online Preview Mode | Online Request Mode | Note                                                                                                                                                                                                                                                                                                                                                                                    |
+|:-----------------------------------------------|--------------|---------------------|---------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [`SELECT Clause`](#selectexprlist)             | **``✓``**    | **``✓``**           | **``✓``**           | A list of projection operations, generally including column names, expressions, or ‘*’ for all columns                                                                                                                                                                                                                                                                                  |
+| [`FROM Clause`](#tablerefs)                    | **``✓``**    | **``✓``**           | **``✓``**           | The From clause indicates the data source.<br />The data source can be one table (`select * from t;`) or multiple tables that JOIN together (`select * from t1 join t2;`) or no table ( `select 1+1;`), see [NO_TABLE SELECT](../dql/NO_TABLE_SELECT_CLAUSE.md)                                                                                                                         |
```

Comment:
From => FROM

Reply:
or multiple tables that JOIN together (`select * from t1 join t2;`)
=>
can you check this statement is supported?

Reply:
capitalize all the SQL keyword, WHERE, GROUP BY, HAVING, WINDOW, ... which are shown in this table

Reply:
> or multiple tables that JOIN together (`select * from t1 join t2;`) => can you check this statement is supported?

it is not supported.

is it ok if I change it to 
 or multiple tables that `LAST JOIN` together (see [JOIN CLAUSE](https://openmldb.ai/docs/en/main/reference/sql/dql/JOIN_CLAUSE.html))